util/chunk/disk_test.go

```diff
@@ -64,30 +64,32 @@ func initChunks(numChk, numRow int) ([]*Chunk, []*types.FieldType) {
 	return chks, fields
 }
 
-func (s *testChunkSuite) TestListInDisk(c *check.C) {
+func TestListInDisk(t *testing.T) {
+	t.Parallel()
+
 	numChk, numRow := 2, 2
 	chks, fields := initChunks(numChk, numRow)
 	l := NewListInDisk(fields)
 	defer func() {
 		err := l.Close()
-		c.Check(err, check.IsNil)
-		c.Check(l.disk, check.NotNil)
+		require.NoError(t, err)
+		require.NotNil(t, l.disk)
 		_, err = os.Stat(l.disk.Name())
-		c.Check(os.IsNotExist(err), check.IsTrue)
+		require.True(t, os.IsNotExist(err))
 	}()
 	for _, chk := range chks {
 		err := l.Add(chk)
-		c.Check(err, check.IsNil)
+		assert.NoError(t, err)
```

Comment:
assert -> require?

Reply:
c.Check = assert, c.Assert = require.
Since they are both used, I didn't change this behavior. 
If you think we should always use require, I can accept it. @mmyj 
 @tisonkun What's your idea?

Reply:
@xxchan I second the opinion to keep the behavior in a migration step.

Reply:
OK